examples/apps/langcensus/src/Display.flix

```diff
@@ -0,0 +1,162 @@
+use Util.Stats
+
+///
+/// The effect to display different analysis results.
+///
+eff Display {
+    def displayByUser(stat: Stats): Unit
+    def displayByRepo(stat: Stats): Unit
+    def displayByByte(stat: Map[String, BigInt]): Unit
+}
+
+mod Display{
+    use Util.Stat
+
+    ///
+    /// Handles the display effect through IO.
+    ///
+    pub def runWithIO(f: Unit -> a \ ef): a \ ef - Display + IO = {
+        run {
+            f()
+        } with handler Display {
+            def displayByRepo(stat, cont) =  {
+                println("\n==================== Analysis Result Per Repo ====================\n");
+                Map.forEach((repo, thisStat) -> {
+                    println("\nRepository: ${repo}");
+                    thisStat |> statFilter |> renameAndMerge |> sortMapByList |> printEachLanguage              
+                }, stat) |> cont
+            }
+            
+            def displayByUser(stats, resume) = {
+                println("\n==================== Analysis Result For User ====================\n");
+                Map.foldLeft((acc, stat) -> {
+                    Map.foldLeftWithKey((acc1, ext, data) -> {
+                        let (adds, dels) = data;
+                        match Map.get(ext, acc) {
+                            case Option.None        => Map.insert(ext, (adds, dels), acc1)
+                            case Option.Some(newData) => let (newAdds, newDels) = newData; Map.insert(ext, (adds + newAdds, dels + newDels), acc1)
+                        }
+                    }, acc, stat)
+                }, Map.empty(), stats) |> statFilter |> renameAndMerge |> sortMapByList |> printEachLanguage |> resume
+            }
+            
+            def displayByByte(stat, resume) = {
+                println("\n==================== Analysis Result By Bytes ====================\n");
+                stat |> sortMapByList |>  List.forEach(data -> {
+                    let (lang, size) = data;
+                    println("${lang}: ${size} Bytes")
+                }) |> resume
+            }
+        }
+    }
+    
+    ///
+    /// Filters out the languages whose adds is zero.
+    ///
+    def statFilter(stat: Stat): Stat = {
+        Map.filter(lines -> {
+            let (adds, _) = lines;
+            adds > 0ii
+        }, stat)
+    }
+        
+    ///
+    /// Prints the stat for a single language.
```

Comment:
This comment does not seem right. It looks like we're going over all the languages, right?